src/audio/tone/Kconfig

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+config COMP_TONE
+	tristate "Tone component"
+	default m if LIBRARY_DEFAULT_MODULAR
+	default y
+	depends on COMP_MODULE_ADAPTER
+	depends on IPC_MAJOR_4
```

Comment:
is it IPC4-only now? But you still have "if IPC3" code in this PR, confused

Reply:
@lyakh only IPC4 is supported now. I left the IPC3 code because Daniel requested to keep it but it doesnt work and needs to be fixed

Reply:
@ranj063 ok, can we then put a comment in Kconfig, explaining, that the dependency on IPC4 is temporary and will be removed once IPC3 support is fixed? OTOH, we could also leave out this dependency and just put a comment here or in CMakeLists.txt that tone is currently broken IPC3 and just not enable it in any configurations that we're using. And if someone enables it - they'll know what they're doing.